softwarereview_editor.es.Rmd

```diff
@@ -70,28 +70,17 @@ Cuando estas fuentes de información no sean suficientes,
 - R-Ladies tiene un [directorio](https://rladies.org/directory/) en el que se especifican las aptitudes e intereses de las personas incluidas en la lista.
 - Puedes publicar la búsqueda en las redes sociales.
 
-##### Consejos para la búsqueda en Airtable {#tips-for-reviewer-search-in-airtable}*Airtable*
+- Puedes tuitear sobre la búsqueda de revisores.
```

Comment:
```suggestion
- Puedes publicar la búsqueda en las redes sociales.
```

Reply:
I suggest we also change this in the English version from
- You might tweet about the reviewer search.
to
- You might post on social media about the reviewer search.